### ISSUE TITLE
Share image not visible on social media

### ISSUE THREAD

**Issue by sai-educ** [sai-educ/society-and-ai#1]
When sharing a link to an article from SAI IRG (any link form societyandai.org) on social, the share image does not appear

**Comment by sai-educ** [comment]
Resources to test for share image:

- Test with LinkedIn Post Inspector: https://www.linkedin.com/post-inspector/
- Test with Facebook Sharing Debugger: https://developers.facebook.com/tools/debug/
- Test with Twitter Card Validator: https://cards-dev.twitter.com/validator
